cluster/src/main/java/com/hartwig/pipeline/execution/vm/ComputeEngine.java

```diff
@@ -174,36 +180,54 @@ private void addNetworkInterface(Instance instance, String projectName) {
         instance.setNetworkInterfaces(singletonList(networkInterface));
     }
 
-    private Image attachDisk(Compute compute, Instance instance, String imageFamily, String projectName, String vmName, long diskSizeGB,
+    private Image attachDisks(Compute compute, Instance instance, String imageFamily, String projectName, String vmName,
             String zone) throws IOException {
         Image sourceImage = resolveLatestImage(compute, imageFamily, projectName);
         AttachedDisk disk = new AttachedDisk();
         disk.setBoot(true);
         disk.setAutoDelete(true);
         AttachedDiskInitializeParams params = new AttachedDiskInitializeParams();
-        params.setDiskSizeGb(diskSizeGB);
         params.setSourceImage(sourceImage.getSelfLink());
         params.setDiskType(format("%s/zones/%s/diskTypes/pd-ssd", apiBaseUrl(projectName), zone));
+        params.setDiskSizeGb(arguments.useScratchSsds() ? 10L : 1000L);
         disk.setInitializeParams(params);
-        instance.setDisks(singletonList(disk));
+        List<AttachedDisk> disks = new ArrayList<>(asList(disk));
+        if (arguments.useScratchSsds()) {
+            attachLocalSsds(disks, projectName, zone);
+        }
+        instance.setDisks(disks);
         compute.instances().attachDisk(projectName, zone, vmName, disk);
         return sourceImage;
     }
 
+    private void attachLocalSsds(List<AttachedDisk> disks, String projectName, String zone) {
```

Comment:
here we use local ssd and not scratch ssd. we should be consistent

Reply:
Yeah agreed. I'm happy with either, I wish they were consistent in the GCP documentation. We can decide and I'll make them consistent.